deltachat-ios/MessageKit/Views/Cells/MediaMessageCell.swift

```diff
@@ -73,12 +81,30 @@ open class MediaMessageCell: MessageContentCell {
             imageView.image = mediaItem.image ?? mediaItem.placeholderImage
             playButtonView.isHidden = true
         case .video(let mediaItem):
-            imageView.image = mediaItem.image ?? mediaItem.placeholderImage
+            if let url = mediaItem.url {
+                if let image = mediaItem.image {
+                    imageView.image = image
+                } else {
+                    // no image in cache
+                    imageView.loadVideoThumbnail(from: url, placeholderImage: mediaItem.placeholderImage, completionHandler: { [weak self] thumbnail in
+                        if let image = thumbnail {
+                            self?.cache(thumbnail: image, key: url.absoluteString)
+                            self?.asyncDelegate?.contentDidLoad()
```

Comment:
this reloads the whole dataset instead of the single cell. It still leads to little interruptions when scrolling. It would be nice if there is a method to request redraw of the UIImageView only. Can you check if there's something like that?

Reply:
Fixed by giving video cells a fixed size -> therefore a reloading is no longer needed.